next.config.js

```diff
@@ -29,7 +29,7 @@ const moduleExports = withBundleAnalyzer(
                 'icons',
                 'modules',
                 'pages',
-                'utils',
+                '@/utils',
```

Comment:
Shouldn't it be just `utils` so it matches other folders? 🤔 They're all on the same level.

Reply:
https://github.com/prezly/theme-nextjs-bea/pull/1213/commits/610b8fc0632e73d0aa79ec3f777daa6d6f249563